src/coprocessor/dag/executor/metrics.rs

```diff
@@ -20,6 +22,7 @@ pub struct ExecutorMetrics {
     pub cf_stats: Statistics,
     pub scan_counter: ScanCounter,
     pub executor_count: ExecCounter,
+    pub runtime_stats: util::RuntimeStats,
```

Comment:
There is another thing to consider. In some executors, it's ExecutorMetrics is just taken from children's, which means that itself doesn't have one actually. Maybe we need some aggregation or vec? But I don't have ideas about how to properly keep orders in the vec (make it the same as executor orders given in protobuf).

Reply:
Is `topn.rs` an example which doesn't contain `ExecutorMetrics`? I find it use `iter` to store the rows but how to get the `ExecutorMetrics` from children's.

Reply:
This is indeed a problem. I will take a look at this next week and try to write a sample implementation for you for table scan so that you can follow